sdk/core/core-http/lib/policies/logPolicy.ts

```diff
@@ -72,39 +72,41 @@ const defaultAllowedHeaderNames = [
   "User-Agent"
 ];
 
-const defaultAllowedQueryParameters: string[] = [
-  "api-version"
-];
+const defaultAllowedQueryParameters: string[] = ["api-version"];
 
 export const DefaultLoggingOptions: LoggingOptions = {
   logger: undefined,
   logPolicyOptions: {
-    allowedHeaderNames: [],      // These are empty lists because they are additive to
-    allowedQueryParameters: []   // the real defaultAllowed[HeaderNames|QueryParameters].
+    allowedHeaderNames: [], // These are empty lists because they are additive to
```

Comment:
The extra spacing here was to make the two comment lines look line a continued sentence, I guess Prettier doesn't appreciate that